src/Instrumentation/AwsSdk/AwsSdkInstrumentation.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Instrumentation\AwsSdk;
+
+use OpenTelemetry\API\Common\Instrumentation\InstrumentationInterface;
+use OpenTelemetry\API\Common\Instrumentation\InstrumentationTrait;
+
+class AwsSdkInstrumentation implements InstrumentationInterface
+{
+    use InstrumentationTrait;
+
+    public function __construct()
```

Comment:
Can you please remove the empty constructor.
Yu can add it againlater, if yu need to pass dependencies (e.g. the AWS SDK)